en/components/toc.yml

```diff
@@ -7,8 +7,21 @@
   href: general/update_guide.md
   new: false
 - name: Ignite UI CLI
-  href: general/cli-overview.md
+  href: general/cli/cli-overview.md
```

Comment:
Well, wouldn't that break the existing CLI topic link?

Reply:
It's moved back to the general root.